consent.py

```diff
@@ -0,0 +1,196 @@
+# encoding: utf-8
+from __future__ import print_function
+import json
+import os
+import urllib
+import urlparse
+import requests
+import xml.etree.ElementTree as elementtree
+from datetime import datetime
+
+import flask
+from flask_seasurf import SeaSurf
+from werkzeug.exceptions import BadRequest
+
+app = flask.Flask('consent')
+
+# Configuration defaults
+app.config['SECRET_KEY'] = None
+app.config['OAUTH_URL'] = 'https://osu-test.apigee.net/oauth2'
+app.config['CAS_URL'] = 'https://login.oregonstate.edu/cas-dev'
```

Comment:
Let's have the prototype consent app failed if the user doesn't provide configuration values.